lib/share.h

```diff
@@ -57,6 +57,7 @@ struct Curl_share {
   struct Curl_ssl_session *sslsession;
   size_t max_ssl_sessions;
   long sessionage;
+  void *known_hosts_shared;
```

Comment:
Everything in this struct is shared, I don't think appending `_shared` to this name here makes much sense.

Reply:
ok corrected